src/libs/actions/IOU.js

```diff
@@ -13,7 +16,94 @@ function getPreferredCurrency() {
     }, 1600);
 }
 
-// Re-enable the prefer-default-export lint when additional functions are added
+function setNewIOUReportObjects(reportIds) {
```

Comment:
Add function doc please with comment.

Maybe `getIOUReportsForNewTransaction` is a bit better?